From 8533e7ec02dd55d939636a7f01a67aa4f2e63547 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 28 Dec 2024 05:34:38 -0800
Subject: [PATCH] Explicitly set version variable

---
 platforms/libretro/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platforms/libretro/Makefile b/platforms/libretro/Makefile
index 0bd2c94..dfb2cae 100644
--- a/platforms/libretro/Makefile
+++ b/platforms/libretro/Makefile
@@ -333,7 +333,7 @@ endif
 
 GIT_VERSION ?= $(shell git describe --abbrev=7 --dirty --always --tags || echo unknown)
 ifneq ($(GIT_VERSION), unknown)
-	CXXFLAGS += -DEMULATOR_BUILD=\"$(GIT_VERSION)\"
+      CXXFLAGS += -DEMULATOR_BUILD=\"3.9.6\"
 endif
 
 include Makefile.common
-- 
2.43.0

