From 4a89fdebd6c97d81c5d1e19cd25c681d36d3d12e Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 28 Dec 2024 05:34:38 -0800
Subject: [PATCH] Explicitly set version variable

---
 platforms/libretro/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platforms/libretro/Makefile b/platforms/libretro/Makefile
index 86ebe18..f5e9a02 100644
--- a/platforms/libretro/Makefile
+++ b/platforms/libretro/Makefile
@@ -331,7 +331,7 @@ endif
 
 GIT_VERSION ?= "$(shell git describe --abbrev=7 --dirty --always --tags || echo unknown)"
 ifneq ($(GIT_VERSION)," unknown")
-	CXXFLAGS += -DEMULATOR_BUILD=\"$(GIT_VERSION)\"
+      CXXFLAGS += -DEMULATOR_BUILD=\"3.5.0\"
 endif
 
 include Makefile.common
-- 
2.43.0

